website/content/docs/agent-and-proxy/autoauth/index.mdx

```diff
@@ -22,7 +22,8 @@ When Vault Agent or Vault Proxy are started with Auto-Auth enabled, it will atte
 Vault token using the configured Method. On failure, it will exponentially back
 off and then retry. On success, unless the auth method is configured to wrap
 the tokens, it will keep the resulting token renewed until renewal is no longer
-allowed or fails, at which point it will attempt to reauthenticate.
+allowed. If renewal fails, the token has been revoked, or the token has exceeded the maximum number of uses,
+it will attempt to reauthenticate.
```

Comment:
```suggestion
allowed. If renewal fails, the token has been revoked, the token has exceeded
the maximum number of uses, or is invalid for any other reason, it will attempt to reauthenticate.
```

Reply:
The 'any other reason' was the initial onus for this request, so I thought I'd call it out explicitly!

Reply:
I just want to make sure that they don't confuse "invalid for any other reason" to be invalid because of invalid policy access.

Reply:
Hmm, you're right: what about "invalid for any reason other than access control"?